src/python/metrics/fuzzer_stats_schema.py

```diff
@@ -363,6 +363,10 @@
     'mode': 'NULLABLE',
     'name': 'strategy_mutator_plugin_radamsa',
     'type': 'INTEGER'
+}, {
+    'mode': 'NULLABLE',
+    'name': 'strategy_peach_grammar_mutation',
+    'type': 'INTEGER'
```

Comment:
would it make sense to record name of the grammar used?

Reply:
+1, that makes more sense as the value here.